src/components/Context/AppContext.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { getTodos } from '../../api/todos';
+import { ContextType } from '../../types/ContextType';
+
+type Props = {
+  children: React.ReactNode,
+};
+
+const AppContext = React.createContext<ContextType | null>(null);
+
+export const AppProvider = ({ children }: Props) => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [filterType, setFilterType] = useState('all');
+  const [loading, setLoading] = useState(false);
+  const [isError, setIsError] = useState('');
```

Comment:
rename variable and create `enum ErrorType`
```suggestion
  const [errorType, setErrorType] = useState('');
```